File: src/main/java/com/dh/ondot/schedule/api/request/PlaceHistoryDeleteRequest.java
package com.dh.ondot.schedule.api.request;

import jakarta.validation.constraints.NotNull;

import java.time.LocalDateTime;

public record PlaceHistoryDeleteRequest(
        @NotNull(message = "검색 시각은 필수입니다.")
        LocalDateTime searchedAt
) {
}

File: src/main/kotlin/com/dh/ondot/schedule/api/request/PlaceHistoryDeleteRequest.kt
package com.dh.ondot.schedule.api.request

import jakarta.validation.constraints.NotNull
import java.time.LocalDateTime

data class PlaceHistoryDeleteRequest(
    @field:NotNull(message = "검색 시각은 필수입니다.")
    val searchedAt: LocalDateTime,
)
